app/src/main/java/app/nexd/android/ui/helper/detail/HelperDetailViewModel.kt

```diff
@@ -57,6 +66,18 @@ class HelperDetailViewModel(private val api: Api) : ViewModel() {
             .blockingSubscribe()
     }
 
+    private fun declineRequest(requestId: Long) {
+        val helpListIdFromRequest =
+            api.helpRequestsControllerGetSingleRequest(requestId).blockingFirst().helpListId
+        helpListIdFromRequest?.let { helpListId ->
+            api.helpListsControllerDeleteHelpRequestFromHelpList(helpListId, requestId)
+                .subscribeOn(io())
```

Comment:
this subscribe on should not be needed because we defined IO as default scheduler for the API client

Reply:
fixed with recent push